Resources/views/Default/admin.html.twig

```diff
@@ -10,13 +10,6 @@
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', {'callback': 'fos.Router.setData'}) }}"></script>
     <script type="text/javascript">
-      //var CSClientId = '{{ clientId }}';
```

Comment:
this shouldn't be done here. This file will be auto-generated on next angular app update.